src/plugins/intel_gpu/src/plugin/transformations/fuse_moe_compressed.hpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/graph_rewrite.hpp"
+
+namespace ov::intel_gpu {
+
+// Fuse subgraph between router and MOECompressed into MOEFusedCompressed operation.
+class FuseMOECompressed: public ov::pass::MatcherPass {
+public:
+    OPENVINO_MATCHER_PASS_RTTI("FuseMOECompressed");
+    FuseMOECompressed();
```

Comment:
This naming is also general, but it is for Gemm3 pattern's target. Please rename this too, for reducing the confusion.